src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -351,6 +352,12 @@ export default class GeFormRenderer extends LightningElement{
         }
     }
 
+    appendToElement(element) {
+        const helper = new GeFormElementHelper(element);
+        element.isRenderable = helper.isRenderable();
+        element.isSourceFieldPicklist = helper.isSourceFieldPicklist();
+    }
```

Comment:
This is where I was envisioning another property called "options" might be added for fields mapped from picklists to checkboxes.